nailgun/entities.py

```diff
@@ -6815,6 +6815,7 @@ def __init__(self, server_config=None, **kwargs):
             'force': entity_fields.BooleanField(default=False),
             'ignore_subman_errors': entity_fields.BooleanField(default=False),
             'repo_data': entity_fields.ListField(),
+            'download_utility': entity_fields.StringField(default=''),
```

Comment:
I would rather go with
```suggestion
            'download_utility': entity_fields.StringField(choices=('curl', 'wget')),
```
since the apidoc says "Must be one of: `curl`, `wget`" and the field is optional (`curl` is used when missing or `None`)

Reply:
Good suggestion, but I think we'll still need to set `default='curl'` when None is passed

Reply:
I tried locally even with `download_utility=None` and it passed, but I agree we can state the default for descriptive reasons here.

Reply:
Done